app/scripts/auth/views/login.html

```diff
@@ -1,19 +1,33 @@
 <form
     novalidate
     ng-submit="login()"
+    name="loginForm"
     class="login-form o-container o-container--narrow u-8/10">
 
   <fieldset class="layout">
     <div class="layout__item u-4/10 u-1/1-palm palm-mb">
       <div class="position--relative">
-        <input type="text" ng-model="user.email" name="email" class="u-1/1" required>
+        <input type="email" ng-model="user.email" name="email" class="u-1/1" required>
         <label for="email">Email</label>
       </div>
+      <div class="error" ng-messages="loginForm.email.$error" role="alert" ng-show="loginForm.email.$touched || (loginForm.$submitted && loginForm.$invalid)">
```

Comment:
this should be refactored to work without ng-show directive. See gist for example: https://gist.github.com/Crawleyprint/2026d76f950198c26706